scanner/src/test/kotlin/storages/CompositeStorageTest.kt

```diff
@@ -82,8 +82,9 @@ private fun licenseFinding(index: Int): LicenseFinding =
 private fun createScanResult(resultCount: Int): ScanResult {
     val licenseFindings = List(resultCount) { licenseFinding(it) }
     val summary = ScanSummary(
-        startTime = Instant.now(), endTime = Instant.now(), fileCount = resultCount,
-        packageVerificationCode = "test$resultCount", licenseFindings = licenseFindings.toSortedSet(),
+        startTime = Instant.now(), endTime = Instant.now(),
+        packageVerificationCode = "test$resultCount",
+        licenseFindings = licenseFindings.toSortedSet(),
```

Comment:
I'm fine with reformatting here, but then we should consistently use one argument per line.